src/components/table/JadeTable.jsx

```diff
@@ -150,82 +106,91 @@ export class JadeTable extends React.PureComponent {
-                            !_.isUndefined(value) && (
-                              <TableCell
-                                key={`${column.id}-${drId}`}
-                                className={classes.cell}
-                                data-cy={`cellvalue-${column.id}-${i}`}
-                              >
-                                {value}
-                              </TableCell>
-                            )
-                          );
-                        })}
-                      </TableRow>
-                    );
-                  })}
-                </TableBody>
-              )}
-            </Table>
-          )}
-          {polling && (
-            <div className={classes.spinWrapper}>
-              <CircularProgress className={classes.spinner} />
-              {delay &&
-                'For large datasets, it can take a few minutes to fetch results from BigQuery. Thank you for your patience.'}
-            </div>
-          )}
-        </div>
-        <TablePagination
-          rowsPerPageOptions={[100]}
-          component="div"
-          count={parseInt(queryResults.totalRows, 10) || 0}
-          rowsPerPage={rowsPerPage}
-          page={page}
-          onChangePage={this.handleChangePage}
-          onChangeRowsPerPage={this.handleChangeRowsPerPage}
-        />
-      </Paper>
-    );
-  }
+  return (
+    <Paper className={classes.root}>
+      <div className={classes.tableWrapper}>
+        {rows && columns && (
+          <Table stickyHeader aria-label="sticky table">
+            <JadeTableHead
+              allowSort={allowSort}
+              columns={columns}
+              createSortHandler={createSortHandler}
+            />
+            {!polling && (
+              <TableBody data-cy="tableBody">
+                {rows.map((row, i) => {
+                  const drId = row.datarepo_id;
+
+                  return (
+                    <TableRow hover tabIndex={-1} key={drId}>
+                      {columns.map((column) => {
+                        const value = handleArrayValues(row[column.id], column);
+                        return (
+                          !_.isUndefined(value) && (
+                            <TableCell
+                              key={`${column.id}-${drId}`}
+                              className={classes.cell}
+                              data-cy={`cellvalue-${column.id}-${i}`}
+                            >
+                              {value}
+                            </TableCell>
+                          )
+                        );
+                      })}
+                    </TableRow>
+                  );
+                })}
+              </TableBody>
+            )}
+          </Table>
+        )}
+        {polling && (
+          <LoadingSpinner
+            delay={delay}
+            delayMessage="For large datasets, it can take a few minutes to fetch results from BigQuery. Thank you for your patience."
```

Comment:
While we're in here, let's maybe take out `from BigQuery` since we're gonna want this for parquet too